actor-mini-profile.js

```diff
@@ -0,0 +1,79 @@
+import { db } from './dbInstance.js'
+
+class ActorMiniProfile extends HTMLElement {
+  static get observedAttributes () {
+    return ['url']
+  }
+
+  constructor () {
+    super()
+    this.url = ''
+  }
+
+  connectedCallback () {
+    this.url = this.getAttribute('url')
+    this.fetchAndRenderActorInfo(this.url)
+  }
+
+  attributeChangedCallback (name, oldValue, newValue) {
+    if (name === 'url' && newValue !== oldValue) {
+      this.url = newValue
+      this.fetchAndRenderActorInfo(this.url)
+    }
+  }
+
+  async fetchAndRenderActorInfo (url) {
+    try {
+      const actorInfo = await db.getActor(url)
+      if (actorInfo) {
+        this.renderActorInfo(actorInfo)
+      }
+    } catch (error) {
+      console.error('Error fetching actor info:', error)
+    }
+  }
+
+  renderActorInfo (actorInfo) {
+    // Clear existing content
+    this.innerHTML = ''
+
+    // Container for the icon and name, which should be clickable
+    const clickableContainer = document.createElement('div')
```

Comment:
For clickable elements, prefer using a `<button>` and removing styles, or if you really need to we must have the correct ARIA roles. If clicking it opens a new page, make it an `a` with an href.